litmus-portal/frontend/src/views/Community/GeoMap/styles.ts

```diff
@@ -29,34 +29,35 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   cityMapComposableMap: {
-    width: '54rem',
-    height: '40rem',
+    width: '45vw',
+    height: '30rem',
+    [theme.breakpoints.down('sm')]: {
+      height: '25rem',
+    },
   },
   countryMapchartStyle: {
     display: 'flex',
     justifyContent: 'center',
     paddingTop: theme.spacing(2),
   },
   countryMapchartContentStyle: {
-    width: '54rem',
-    height: '40rem',
+    width: '45vw',
+    height: '30rem',
```

Comment:
![Screenshot from 2021-03-12 18-24-55](https://user-images.githubusercontent.com/40641427/110942765-4a34b780-8360-11eb-9a63-c45b82d8efeb.png)


using vw will cause such a screen ^^ below 1920x1080p monitors (1366x768) are still very much in use, in any case, there is no need to manually control the width for the map as it isn't responsive either way

the breakpoint also doesn't help here as we are anyways not making the maps responsive and it will result in a map like this at breakpoint 'sm' that triggers at screen widths below 960px:

![Screenshot from 2021-03-12 18-28-34](https://user-images.githubusercontent.com/40641427/110943180-ea8adc00-8360-11eb-8367-e99fe7fdc4c1.png)

The paper on which we are trying to render the map and the actual map itself are on different styles so there is overlapping, it's best to not control the width with any styles at all which was the reason I had originally chosen to write it that way:

![Screenshot from 2021-03-12 18-31-26](https://user-images.githubusercontent.com/40641427/110943420-30e03b00-8361-11eb-8e6a-fbe083978aa7.png)

Unless I'm missing something it would be better to remove this and the `cityMapComposableMap` style entirely

Reply:
yes, I have tested on real monitors on 3 screens, this becomes responsive only when using `vw` @vanshBhatia-A4k9 its not possible to test the changes with dev tools I believe because, on the real screen, it looks correctly resized.

Reply:
will attach some screenshots for refernce

Reply:
I don't believe we are making the pages responsive anyways because we don't have those designs, so I don't see the requirement & simply removing these changes will do , that being said the devtools are there to test responsiveness only, I'm not sure how it is possible that these are giving the wrong output

Reply:
Don't use vw if not required...here it is not required

Reply:
In that case, if your screen resolution is good you may pull the changes from the branch and test locally to check the changes, picking selective changes won't work.